projects/docs/docs/commands/test.md

```diff
@@ -57,3 +57,4 @@ One of the benefits of using Tuist over other automation tools is that developer
 | `--device`          | `-d`  | `Test on a specific device.`                                        |                   | No       |
 | `--os`              | `-o`  | `Test with a specific version of the OS.`                           |                   | No       |
 | `--configuration`   | `-C`  | `The configuration to be used when building the scheme.`            |                   | No       |
+| `--skip-ui-tests`   | `-C`  | `When passed, it skips testing UI Tests targets.`                   | False             | No       |
```

Comment:
I believe the short flag here is wrong.

Reply:
Fixed, by not allowing a short. The reason behind it mentioned [here](https://github.com/tuist/tuist/pull/2832#discussion_r631565504).